src/test-harness/SparkEthereumTests.sol

```diff
@@ -311,4 +316,142 @@ abstract contract SparkEthereumTests is SparklendTests {
     ) internal view {
         _assertMorphoCap(_vault, _config, _currentCap, false, 0);
     }
+
+    /******************************************************************************************************************/
+    /*** Internal testing helper funcitons                                                                         ****/
+    /******************************************************************************************************************/
+
+    struct CollateralOnboardingTestParams {
+        // General
+        string  symbol;
+        address tokenAddress;
+        uint256 tokenDecimals;
+        address oracleAddress;
+        // IRM Params
+        uint256 optimalUsageRatio;
+        uint256 baseVariableBorrowRate;
+        uint256 variableRateSlope1;
+        uint256 variableRateSlope2;
+        // Borrowing configuration
+        bool borrowEnabled;
+        bool stableBorrowEnabled;
```

Comment:
Can remove this and always assume false.

Reply:
As per convo in slack leaving in and hardcoding to false in test